### PR TITLE
Add vprtempo package to conda-forge

### PR DESCRIPTION
**Pull request by AdamDHines**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by AdamDHines** [comment]
@conda-forge/help-python, ready for review!

**Comment by AdamDHines** [comment]
@conda-forge-admin, please ping team

File: recipes/VPRTempo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/VPRTempo-{{ version }}.tar.gz
+  sha256: 2660af11c463893b22353967e7fc14b502b3b6d86331d66f47e33a43f11519aa
+
+build:
+  skip: true  # [win]
```
Comment (xhochy) [inline recipes/VPRTempo/meta.yaml:13]: Looks like this should be noarch
```suggestion
  noarch: python
```

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/VPRTempo/meta.yaml (hunk 1 above)
Comment (xhochy) [reply recipes/VPRTempo/meta.yaml:13]: Failing windows CI because of missing dependencies is OK.

File: recipes/VPRTempo/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  sha256: 2660af11c463893b22353967e7fc14b502b3b6d86331d66f47e33a43f11519aa
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
```
Comment (xhochy) [inline recipes/VPRTempo/meta.yaml:19]: This should be in sync.
```suggestion
    - python >=3.6
```

Comment (AdamDHines) [reply recipes/VPRTempo/meta.yaml]: Thanks fixed it all up, should be good now

**Review by AdamDHines — COMMENTED** [review]
All fixed

**Review by xhochy — APPROVED** [review]

**Pull request merged**